sycl/CMakeLists.txt

```diff
@@ -12,10 +12,10 @@ option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 include(AddSYCLExecutable)
 
-set(SYCL_MAJOR_VERSION 3)
+set(SYCL_MAJOR_VERSION 4)
```

Comment:
There are no ABI breaking changes here, so `SYCL_MAJOR_VERSION` should stay 3.
See https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md#abi-versioning-policy

Reply:
Done. Changed to `3.1` as minor version says about non-breaking changes.

Reply:
@bader, good catch!!